providers/prov_running.c

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/e_os2.h>
+#include "prov/providercommon.h"
+
+int provider_is_running(void)
+{
+    return 1;
```

Comment:
Should this raise an error - or is that problematic ?

Reply:
The FIPS equivalent could raise an error if the provider isn't happy.  The error should have been be reported when it happened, does it need to be re-reported each time it stops an operation -- I don't know the answer, both sides have merit.